include/wx/zipstrm.h

```diff
@@ -225,17 +225,19 @@ class WXDLLIMPEXP_BASE wxZipEntry : public wxArchiveEntry
     wxArchiveEntry* DoClone() const wxOVERRIDE             { return ZipClone(); }
 
     size_t ReadLocal(wxInputStream& stream, wxMBConv& conv);
-    size_t WriteLocal(wxOutputStream& stream, wxMBConv& conv) const;
+    size_t WriteLocal(wxOutputStream& stream, wxMBConv& conv, bool checkForUTF8) const;
```

Comment:
As always, I'd prefer to use something like `int flags = Flags_UseUTF8IfNecessary` instead of `bool checkForUTF8`, but this is relatively minor for this non-public function.